pom.xml

```diff
@@ -33,6 +33,7 @@
         <jmock.version>2.6.0</jmock.version>
         <slf4j.log4j.version>1.7.5</slf4j.log4j.version>
         <agrona.version>0.3.1</agrona.version>
+        <metrics.version>develop-SNAPSHOT</metrics.version>
```

Comment:
All we need do is make sure I emerged the metrics upon request first, then we release metrics, then I can merge this request.